lib/menu.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'item'
+require 'csv'
+
+class Menu
+
+  attr_reader :list
+
+  def initialize
+    @list = []
+  end
+
+  def add(name, price, item = Item.new(name, price))
+    @list << item
+  end
```

Comment:
Are you adding the item as a parameter for mocking in your tests? It feels a bit out of place here, you may want to consider doing dependency injection of the class Item in your constructor.